stable/node-problem-detector/templates/daemonset.yaml

```diff
@@ -31,7 +31,7 @@ spec:
           command:
             - "/bin/sh"
             - "-c"
-            - "exec /node-problem-detector --logtostderr --system-log-monitors={{- range $index, $monitor := .Values.settings.log_monitors }}{{if ne $index 0}},{{end}}{{ $monitor }}{{- end }}"
+            - "exec /node-problem-detector --logtostderr --system-log-monitors={{- range $index, $monitor := .Values.settings.log_monitors }}{{if ne $index 0}},{{end}}{{ $monitor }}{{- end }} --custom-plugin-monitors={{- range $index, $monitor := .Values.settings.custom_plugin_monitors }}{{if ne $index 0}},{{end}}{{ $monitor }}{{- end }}"
```

Comment:
How is this going to work by default when `settings. custom_plugin_monitors` is just an empty map? Won't npd be run with the `--custom-plugin-monitors=` argument but with no value?

Reply:
I have tried fixing this with:
```
{- if .Values.settings.custom_plugin_monitors }} ... {{- end}}
```

Please let me know if that is adequate.

Reply:
@Hashfyre please avoid force pushing as it prevents us seeing the changes you make after the review. Can you also address the other comment I made about the README?

Reply:
Yes, fixed that. Will keep the no force-push policy in mind.